src/themes/shared/typography.js

```diff
@@ -13,221 +13,152 @@ export const fontTokens: FontTokensT = {
-  },
-  ParagraphXSmall: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '12px',
-    fontWeight: 'normal',
-    lineHeight: '20px',
-  },
-  ParagraphSmall: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '14px',
-    fontWeight: 'normal',
-    lineHeight: '20px',
-  },
-  ParagraphMedium: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '16px',
-    fontWeight: 'normal',
-    lineHeight: '24px',
-  },
-  ParagraphLarge: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '18px',
-    fontWeight: 'normal',
-    lineHeight: '28px',
-  },
-  LabelXSmall: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '12px',
-    fontWeight: 500,
-    lineHeight: '20px',
-  },
-  LabelSmall: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '14px',
-    fontWeight: 500,
-    lineHeight: '20px',
-  },
-  LabelMedium: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '16px',
-    fontWeight: 500,
-    lineHeight: '24px',
-  },
-  LabelLarge: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '18px',
-    fontWeight: 500,
-    lineHeight: '28px',
-  },
-  HeadingXSmall: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '20px',
-    fontWeight: 500,
-    lineHeight: '28px',
-  },
-  HeadingSmall: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '24px',
-    fontWeight: 500,
-    lineHeight: '32px',
-  },
-  HeadingMedium: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '28px',
-    fontWeight: 500,
-    lineHeight: '36px',
-  },
-  HeadingLarge: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '32px',
-    fontWeight: 500,
-    lineHeight: '40px',
-  },
-  HeadingXLarge: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '36px',
-    fontWeight: 500,
-    lineHeight: '44px',
-  },
-  HeadingXXLarge: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '40px',
-    fontWeight: 500,
-    lineHeight: '52px',
-  },
-  DisplayXSmall: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '36px',
-    fontWeight: 500,
-    lineHeight: '44px',
-  },
-  DisplaySmall: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '44px',
-    fontWeight: 500,
-    lineHeight: '52px',
-  },
-  DisplayMedium: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '52px',
-    fontWeight: 500,
-    lineHeight: '64px',
-  },
-  DisplayLarge: {
-    fontFamily: themePrimitives.primaryFontFamily,
-    fontSize: '96px',
-    fontWeight: 500,
-    lineHeight: '112px',
-  },
-});
+  };
+
+  return {
+    font100,
+    font150,
+    font200,
+    font250,
+    font300,
+    font350,
+    font400,
+    font450,
+    font550,
+    font650,
+    font750,
+    font850,
+    font950,
+    font1050,
+    font1150,
+    font1250,
+    font1350,
+    font1450,
+    ParagraphXSmall: font100,
```

Comment:
I recommend reversing this and using the new names as the default, and setting the fontXXX using the default names. I assumed that in the future the old names might be removed.

Reply:
just discussed offline - probably we could keep both, and get away without a breaking change here in the future too (these are just aliases, so there is a barely any impact on the bundle size)